assets/js/components/link.js

```diff
@@ -42,6 +42,11 @@ class Link extends Component {
 			id,
 		} = this.props;
 		const SemanticLink = href ? 'a' : 'button';
+		// Note: the disabled attribute does not alter behavior of anchor tags,
+		// hence the conditional click handler.
+		const clickHandler = disabled ?
+			( e ) => e.preventDefault() :
```

Comment:
This click handler by itself is not good for accessibility. Assistive technologies will read it as a link and users will wonder why the link is not working.

Yes, `disabled` does not work on links, but `aria-disabled` does.

We could even take this further and use `pointer-events: none` and different cursors.

See https://css-tricks.com/how-to-disable-links/ for an intro.

Reply:
That's a good point. Now that I think about it, in this case, I think we can just force it to be a `button` instead since the `href` would be pointless on a disabled link anyways.
That way no change should be needed to the click handler (which should probably only be added if it's a button anyways?)

> Yes, `disabled` does not work on links, but `aria-disabled` does.

`aria-disabled` only describes the disabled state for screen readers, it doesn't affect the behavior. In the linked article, the author uses the attribute to apply the `pointer-events: none` rule which implements the click disabling.

![image](https://user-images.githubusercontent.com/1621608/65413861-16c43d00-ddfb-11e9-89b6-c9f60b10e38e.png)

It wouldn't be a bad idea to use the aria attribute here but I would imagine screen readers should understand `button[disabled]` since that is a standard.

What do you think?